examples/aliens.py

```diff
@@ -304,7 +304,6 @@ def main(winstyle=0):
 
     # Run our main loop whilst the player is alive.
     while player.alive():
-
```

Comment:
There's enough of these random empty line removals/additions that I think deserves another PR, but do not belong in this one because they are completely unrelated to `_sdl2.Window`